### PR TITLE
Feature/address and icons

### PR DESCRIPTION
**Pull request by mmorey**

### CODE REVIEW

**Comment by mmorey** [comment]
Hello @tneschke! Welcome to the team!

Overall this all looks really good, nice job. I see 2 minor issue that needs to be fixed:

The App Survey link you replaced is actually used in 2 places which is why it was defined as a constant `Constants.appSurveyURL`. Take a look at `AppCordinator.swift` to see the other location. I would recommend moving the hard coded URL into the `Constants.swift` file so that it's fixed in both places.

On the map view, if you zoom out, it seems one of the icons you replaced is massive and takes over the whole screen. I'm seeing this issue on the simulator and on an actual device. Here are some screenshots showing the issue:

IMG_1468
IMG_1467

File: Endangered Waves/information/InformationViewController.swift [hunk 1]
```diff
@@ -109,7 +112,7 @@ extension InformationViewController {
         case 3:
             switch indexPath.row {
             case 0: // App Survey
-                let url  = URL(string: Constants.appSurveyURL)!
+                let url  = URL(string: "https://www.savethewaves.org/appsurvey/")!
```
Comment (mmorey) [inline Endangered Waves/information/InformationViewController.swift]: This URL needs to change in `Constants.swift` as it impacts `AppCordinator.swift` as well.

Comment (tneschke) [reply Endangered Waves/information/InformationViewController.swift]: Hello @mmorey,

Thanks for suggesting the survey url fix. I've edited that, and also fixed the oversized icon display.

That was present before my changes, so I assumed it was from an incorrect sized file in firebase. This is for the world surfing reserve in Playa Hermosa. I fixed the icon by resizing it to 48px, but I'm not sure if that's necessary with clean data. That specific icon still displays incorrectly as transparent and without a border. 

simulator_screenshot_01BF4B8B-7269-476C-941B-C3FF8EDB5BBD

**Comment by mmorey** [comment]
@tneschke good call out, I didn't think the issue could be with the asset on Firebase, @diego might have updated it but didn't use a proper size. Nice use of `SDImageResizingTransformer()`, I didn't know about that feature of `SDWebImage`.

Everything looks good from what I can see. Are you planning on adding more features before a new build goes on the App Store?

If your done with features go ahead and bump the version, I'm guessing 2.4.1 should work.

**Comment by tneschke** [comment]
> @tneschke good call out, I didn't think the issue could be with the asset on Firebase, @diego might have updated it but didn't use a proper size. Nice use of `SDImageResizingTransformer()`, I didn't know about that feature of `SDWebImage`.
> 
> Everything looks good from what I can see. Are you planning on adding more features before a new build goes on the App Store?
> 
> If your done with features go ahead and bump the version, I'm guessing 2.4.1 should work.


Thanks!

I think they also wanted me to try and fix some of the crash instances before releasing an update, so I can let you know. 

Thanks for taking the time to review this, much appreciated!

**Comment by mmorey** [comment]
@tneschke this is all looking good. Your last comment mentions the "image upload crash". Where in the code did you make changes that fix that? Its not obvious to me?

**Comment by tneschke** [comment]
> 



> @tneschke this is all looking good. Your last comment mentions the "image upload crash". Where in the code did you make changes that fix that? Its not obvious to me?

In the image carousel for submitting reports if the image isn't properly loaded (like the 6th image in the simulator), it will crash due to accessing an empty array. So I added "if !images.isEmpty" to ImageSliderViewController.swift

**Comment by mmorey** [comment]
Looks good, merging now.

**Pull request merged**